ivy/functional/frontends/jax/nn/non_linear_activations.py

```diff
@@ -98,19 +88,29 @@ def _reduction_dims(a, axis):
     else:
         return canon_axis, canon_axis
 
+# Helpers #
+# ------- #
```

Comment:
If we are going to include 
```
# Helpers #
# ------- #
```
it should be before the first helper and we should also have another header after the last helper, smth like
```
# API Functions #
# ------- #
```